tests/unit/test_oidc.py

```diff
@@ -0,0 +1,226 @@
+"""
+Unit tests for the oidc code in the frontend.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+import ga4gh.frontend as frontend
+import tests.utils as utils
+from mock import patch
```

Comment:
Split up these imports: third party modules go into a group after the standard lib, and local imports go last.